File: HW2/1.rb
# Tehnologichno uchilishte Elektronni sistemi kym TU Sofiq.
# 11 A klas
# Nomer 15
# Liuboslav Dimitrov Jordanov
# Prochita razlichnite vidove skriptovi ezici ot edna kolonka ot CSV fail i izvejda broq im.




require 'csv'

def process_file_with_csv
	f = File.open("/home/student11a/11a/klasa.csv", 'r')
	rub=0
	pyth=0
	per=0
	content = f.read
	result=0
	content.each_line do |line|
		array = line.split(",")
		if array[4] == "Ruby"
			rub=rub+1
		elsif array[4] == "Python" 
			pyth=pyth+1
		elsif array[4] == "Perl"
			per=per+1
		else
			print "nqma vuveden skriptov ezik"
		end
	print rub
	print pyth
	print per
		result = result + array[4].to_i
	end
	p result
end

	process_file

File: HW1/1.cc
//Tehnologichno uchilishte Elektronni sistemi kym TU Sofiq.
//11 A klas
//Nomer 15
//Liuboslav Dimitrov Jordanov
//Namira i izvejda chislata, delqshti se na 17 v intervala ot 2 drugi chisla.




# include <stdio.h>

int main()
{
      int x, y, n, i;
      printf("vuvedi na4alo", x);
      scanf("%d",&x);
      printf("vuvedi krai", y);
      scanf("%d",&y);
      
      
      n = x;
      
      while (n<=y){
            if (n%17 == 0)
            i = i + n;
            else {
                 n = n++;
                 }
                 }
      printf("%d",i);
            
      return 0;        
}
